evalml/tests/component_tests/test_utils.py

```diff
@@ -29,13 +29,13 @@ def test_all_components(
     has_minimal_dependencies, is_running_py_39_or_above, is_using_conda
 ):
     if has_minimal_dependencies:
-        n_components = 37
+        n_components = 39
     elif is_using_conda:
-        n_components = 48
+        n_components = 50
     elif is_running_py_39_or_above:
-        n_components = 47
-    else:
         n_components = 49
```

Comment:
Kind of confused as to why this isn't throwing a codecov warning for lack of coverage. Did Codecov not check this change because `n_components = 49` seems to have come from the `else` statement before? That's wild